src/frontend/src/api/index.js

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+
+const config = {
+    baseUrl: 'http://localhost:8080/api/users/'
+};
```

Comment:
```suggestion
const config = {
    baseUrl: '/api/users/'
};
```
이렇게 하시면 CORS 문제 안생길거예요.
vue config에 proxy 설정을 해둬서 자동으로 8080으로 갑니다!